docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+When using formal names, use them fully spelled out and capitalized, without further formatting. Of course, the context of use should be OSCAL object types (which are defined with such formal names), not something else. In this context, in other words, they are meaningful as proper nouns.
+
+A complete list of formal names for objects defined in OSCAL schemas can be retrieved using the XPath query `//formal-name` (binding unprefixed names to namespace `http://csrc.nist.gov/ns/oscal/1.0`), over the collection of OSCAL metaschemas.
+
+#### Other OSCAL and non-OSCAL terminology
+
+In general, technical terms (and even formal names) should be avoided in favor of non-technical language except when necessary for specific clarification *within the context of use* including your intended audience. For instance, a tutorial intended for beginners does not (and should not) exhaust every topic: it should be technical only as appropriate and necessary, and technical language not actually useful to understand problems or concepts at hand, can often be saved for elsewhere.
+
+Nonetheless, technical terminology is often necessary. For documents reaching non-technical audiences (or non-domain experts), this often requires offering brief definitions for terms at the head of a document, or even a glossary. Even technical audiences can be helped by a link to a specification that defines formal terms. In any case, formal terminologies can be called out and identified as such. For example, both OSCAL and RMF (Risk Management Framework) terminology may combine in a description of a use case for a new feature. In this instance a brief notice of what is meant by terms such as <q>profile</q> or <q>baseline</q> is not out of order.
+
+With the same objective of lucidity and lightness, a document intended for a specific audience should not have to define terms for other audiences. A `readme` directed at Python developers is free to use terms appropriate to Python.
+
+## NIST data publishing
+
+In addition to maintaining web platforms, the OSCAL project publishes resources and tools through the formal NIST data publishing process, for release through NIST's [Science Data Portal](http://data.nist.gov). This is appropriate for (NIST-authored) contributions that should be indexed as scientific publications, or archived as discrete data sets.
+
+The formal review process this entails will presumably not be a problem for works the site editors are also reviewing. Mechanically, the process can be lightweight, requiring nothing more than a structured metadata record with a link to a published (web-accessible) OSCAL resource. This record is provided and maintained in NIST's metadata repository for [Management of Institutional Data Assets](https://midas.nist.gov/)). 
```

Comment:
I do not believe MIDAS is a public resource accessible outside of NIST staff's intranet, so do we reference it? I know we should remind people there is a data management policy, but Google dorking to search for it outside of NIST intranet I struggle to find mention of it. I find a lot about another scientific project named MIDAS though.

Reply:
Edited to remove the links requiring credentials. Readers will just have to follow instructions on finding them.